src/components/institution_table/InstitutionTable.js

```diff
@@ -72,7 +72,7 @@ export default function InstitutionTable(props) {
   return (
     div({className: 'institution-table-component'}, [
       div({style: Styles.TABLE.CONTAINER}, [
-        div({style: Styles.TABLE.HEADER_ROW}, [tableHeaderTemplate]),
+        div({style: Styles.TABLE.HEADER_ROW}, tableHeaderTemplate),
```

Comment:
this line fixes the "unique key in table" error that shows up when you click on the admin institution tab